### PR TITLE
RATIS-2098. RaftLogTruncateTests may fail.

### PR DESCRIPTION
**Pull request by szetszwo**
See RATIS-2098.

### CODE REVIEW

**Comment by szetszwo** [comment]
Passed 100 times https://github.com/szetszwo/ratis/actions/runs/9184276839

**Comment by szetszwo** [comment]
With out this change, the test does fail on the master branch https://github.com/szetszwo/ratis/actions/runs/9185440159

**Review by adoroszlai — COMMENTED** [review]
Thanks @szetszwo for the patch.  LGTM, just a minor change in log may be missing.

File: ratis-test/src/test/java/org/apache/ratis/RaftLogTruncateTests.java [hunk 1]
```diff
@@ -178,9 +179,11 @@ void runTestLogTruncate(MiniRaftCluster cluster, RaftServer.Division oldLeader,
       LOG.info("done");
     }
 
-    // kill the old leader
+    // kill remaining peers
     LOG.info("Before killServer {}: {}", oldLeader.getId(), cluster.printServers());
```
Comment (adoroszlai) [inline ratis-test/src/test/java/org/apache/ratis/RaftLogTruncateTests.java:183]: Match `After killServer` message?

```suggestion
    LOG.info("Before killServer {}: {}", remainingPeers, cluster.printServers());
```

**Comment by szetszwo** [comment]
@adoroszlai , thanks for the review!  I have applied your suggestion.

**Review by adoroszlai — APPROVED** [review]

**Comment by adoroszlai** [comment]
Thanks @szetszwo for updating the patch.

**Pull request merged**